src/VirtualList/VirtualList.js

```diff
@@ -20,25 +20,30 @@ class VirtualList extends React.Component {
 
     this.state = this.getVirtualState(this.props);
 
-    // Replace onScroll by debouncing
+    // Replace onScroll by throttling
     if (this.props.scrollDelay > 0) {
       this.onScroll = Util.throttle(
         this.onScroll.bind(this),
-        this.props.scrollDelay
+        this.props.scrollDelay,
+        // Fire on both leading and trailing edge to minize flash of
+        // un-rendered items
+        {leading: true, trailing: true}
```

Comment:
Reading your documentation of the respective options I assumed the functions will be executed twice in a very short time as illustrated below, but the leading/trailing actually only refers to the every first and last call thus we're good.

```
| wait | wait | wait |
 ^    ^ ^    ^ ^    ^
```